frontend/src/index.tsx

```diff
@@ -8,6 +8,10 @@ import { App } from './app/App'
 import reportWebVitals from './reportWebVitals'
 import * as serviceWorker from './serviceWorker'
 
+if (process.env.NODE_ENV === 'development') {
```

Comment:
how does this env var get injected into the frontend? currently the setup is such that `docker-compose` will inject all the required env vars in the dev environment even if there's no `.env` - will this behaviour be preserved?

also should this be `test` instead of `development`? we don't want API calls to be mocked for development right?

Reply:
Moved to `test` environment by default